src/App.js

```diff
@@ -189,13 +225,15 @@ class App extends React.Component {
                   />
                   <div className={`share__container ${isOpen === 3 ? '' : 'hidden'} collapsibles`}>
                   
-                    <button className="share-button" type="button">
+                    <button className="share-button" type="button" onClick={this.sendData}>
                       <i className="far fa-address-card" /> Crear tarjeta
                     </button>
-                    <section className='section__twitter collapsible__hidden'>
+                    <section className='section__twitter '>
+                    {/* collapsible__hidden */}
```

Comment:
This class should be active in the twitter section, so when the url is loaded (has info) it is removed and the panel is visible. I think it could be something like that:
```<section className=`section__twitter ${this.state.url ? '' : 'collapsible__hidden'}`>```
This would come to say that if `this.state.url` is true (has data) do not put me class (so the panel will be shown) otherwise (false) keep the class `collapsible__hidden`
I hope I've explained, otherwise let me know and I'll explain in person :)